flask_project/url_scanner.py

```diff
@@ -31,3 +33,66 @@ def get_domain(url: str) -> str:
     """
     parsed_url = urlparse(url)
     return parsed_url.netloc
+
+def url_validation(url):
+    """
+    Function check if given url name has correct URL structure
+
+    :Parameters:
+        - url (string): String with URL to check
+    :Return:
+        - True/False
+    """
+    validation = urlparse(url)
+    return bool(validation.scheme and validation.netloc and validation.scheme in ['http', 'https'])
```

Comment:
Is it really necessary to convert result to boolean type explicitly?

Reply:
Last line will give empty string when checked variable is not correct url address, that is the reason why I used boolen conversion on beginning.